### PR TITLE
#167401602  Setup development environment for the API

### PR DESCRIPTION
**Pull request by nkalyesubula**
#### What does this PR do?
- It's intended to set up the development environment for the API. It includes all the necessary files for starting the API.
#### How can this be tested?
 - Clone the repo and **cd** in to andaapp and then type: **npm run dev**
#### Appropriate user story id
 - [#167401602](https://www.pivotaltracker.com/story/show/167401602)

### CODE REVIEW

**Comment by nshutijonathan** [comment]
- You have pushed node_modules to Github, It's better to ignore them

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - postgresql
+
+env:
+  global:
+    - NODE_ENV=test
+
+before_script:
+  - psql -c 'create database andaapp_test;' -U postgres
+  - psql -c "CREATE USER noahkalyesubula WITH PASSWORD null;" -U postgres
+  - npm run build
```
Comment (minega25) [inline .travis.yml:19]: Why are we building the app at this point?

File: . Procfile [hunk 2]
```diff
@@ -0,0 +1 @@
+web: node src/app.js
```
Comment (minega25) [inline . Procfile]: In production, its better if we run the es5 version of our app in `/dist` forlder

**Comment by Anguandia** [comment]
Hi, good work! I think though we need to verify if the env file is supposed to be pushed to version control

**Review by minega25 — CHANGES_REQUESTED** [review]
This is great work. I have included the comments in my review. Lets use .gitignore for some folders like node_modules and files

**Pull request merged**